data/templates/js/collapse-infotable.js

```diff
@@ -1,4 +1,10 @@
 $(function() {
+        // Make sure we only have one div for ordering of the elements
+        var divP = $(".mw-parser-output");
```

Comment:
It looks like `collapse_infotable.js` is run on non-server templated articles in the companion app too: https://github.com/endlessm/eos-companion-app-integration/blob/master/eoscompanion/ekn_content_adjuster.py#L193

Perhaps it would be good to have it only run on wikipedia articles from within the renderer itself, eg by adding a source type check to https://github.com/endlessm/eos-knowledge-content-renderer/blob/master/eknrenderer/eknr-renderer.c#L509 . You would have to put a JS-level media query in place to ensure that it only runs on phones, though.

Reply:
About the Wikipedia articles, it is actually the other way around: mw-parser-output is present in SDK4 apps an not in SDK3 ones. We move the first paragraph around (to appear at the top) not the infobox. There are several infoboxes as well, so it could get hairy to move the infobox.

Reply:
@erikos was your comment just here in response to @ptomato review-level comment above about Wikipedia's rendered output?

Reply:
Yes, just for the idea to move the infobox.